index.html

```diff
@@ -0,0 +1,236 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>chigozie ezeilo portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <a> <div class="wrapper">
+        <header>
+            <nav>
+                 <img src="./images/Logo Placeholder.svg" alt="" class="logo">
+                </a>
+                 <button class="menu"><i class="fafa-bars"></i></button>
+             </nav>
+        </header>
+        <section class="header-container">
+            <h1>Hey There. I'm chigozie</h1>
+            <h3>I'm a software developer</h3>
+            <p>i can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don't hestiate to contact me.</p>
+
+                <ul class="social-container">
+                    <li><a href="#"><img src="./images/git.svg" alt=""></a></li>
+                    <li><a href="#"><img src="./images/linkedin.svg" alt=""></a></li>
+                    <li><a href="#"><img src="./images/v.svg" alt=""></a></li>
+                    <li><a href="#"><img src="./images/twitter.svg" alt=""></a></li>
+                    <li><a href="#"><img src="./images/M.svg" alt=""></a></li> 
+                </ul>
+
+
+        </section>
+        <section class="work-container">
+            <h2 class="recent-work">My Recent Work</h2>
+            <div class="featured-card">
+                <img src="./images/featured.svg" alt="" class="featured">
+                <div class="multi-container">
+                    <h3 class="feature">Multi-Post Stories</h3>
+                    <p class="feature">A daily selection of privately personalized reads; no accounts or sign-ups
+                        required. has been the industry's standard dummy text ever since the 1500s, when an unknown
+                        printer took a standard dummy text.
+                    </p>
+                        <ul class="feature-tags">
+                            <li>CSS</li>
+                            <li>html</li>
+                            <li>bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                        <button class="btn">See Project</button>
```

Comment:
- [ ] You have done a great job adding the hover effects to your buttons 👏🏽, However, the `See Project` button for your first project looks a little bit off from what you have in the given design 🔨. Please consider styling your button to match the design 👇🏽.

### _Your design_
![image](https://user-images.githubusercontent.com/43172164/199295925-58591132-821c-4ebb-b5f3-daa6423faba9.png)

### _Figma design_
![image](https://user-images.githubusercontent.com/43172164/199296096-6d658de5-605c-47d4-a2b3-d0d259a1b527.png)